examples/api/lib/widgets/sliver/sliver_coordinator.0.dart

```diff
@@ -0,0 +1,157 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+void main() {
+  runApp(const SliverCoordinatorExampleApp());
+}
+
+class SliverCoordinatorExampleApp extends StatelessWidget {
+  const SliverCoordinatorExampleApp({ super.key });
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(home: SliverCoordinatorExample());
+  }
+}
+
+class SliverCoordinatorExample extends StatefulWidget {
+  const SliverCoordinatorExample({ super.key });
+
+  @override
+  State<SliverCoordinatorExample> createState() => _SliverCoordinatorExampleState();
+}
+
+class _SliverCoordinatorExampleState extends State<SliverCoordinatorExample> {
+  static const String alignedItemId = 'alignedItem';
+  late final ScrollController scrollController;
+
+  @override
+  void initState() {
+    super.initState();
+    scrollController = ScrollController();
+  }
+
+  @override
+  void dispose() {
+    scrollController.dispose();
+    super.dispose();
+  }
+
+  void autoScrollTo(double offset) {
+    scrollController.position.animateTo(
+      offset,
+      duration: const Duration(milliseconds: 300),
+      curve: Curves.easeInOut,
+    );
+  }
+
+  // Called each time a scroll gesture ends. If the alignedItem overlaps
```

Comment:
```suggestion
  // Called each time a scroll gesture ends. If the alignedItem is only partially visible
```

Is this what you mean by overlap here? Or is it to do with one sliver overlapping another?

Reply:
That's what I meant by "alignedItem overlaps either end of the CustomScrollView's viewport", yes. Just saying "partially visible" is ambiguous since there are lots of other reasons an item might be partially visible. Will weave "visible" in there just to make it clearer.